doc/switch-components.md

```diff
@@ -0,0 +1,55 @@
+# Controlled Switch Component
+https://reactjs.org/docs/forms.html#controlled-components
+
+- Our custom switch component substituting reactstrap's <CustomInput type="switch"/>
+- The input’s value is driven by the React state
+- parameters
+    - isOn (boolean) - sets state
+    - toggle (function) - triggered upon click
+    - title (string)
+    - disabled (boolean / optional)
+
+- example:
+<!-- 
+    import React, { useState } from 'react'
+    import {ControlledSwitch} from 'asab-webui'
+
+    const YourComponent = (props) => {
+	    const [isOn, setIsOn] = useState(false)
+		return (
+			<ControlledSwitch  toggle={() => setIsOn(!isOn)} title='setTotp' isOn={isOn}/>
+		)
+    } 
+-->
+
+,,,,,
+
+# Uncontrolled Switch Component
+https://reactjs.org/docs/uncontrolled-components.html#gatsby-focus-wrapper
+
+- Form data is handled by the DOM itself
+
+- parameters
+	title () - switcher's title
+    register () - method allowing you to register an input
+    setValue () -  function allowing you to dynamically set the value of a registered field
+    name - registered field's name (key)
+    id 
+	defaultValue (optional)
+    disabled (optional)
+
+- example:
+    <!-- 
+    import React from 'react'
+    import { UncontrolledSwitch } from 'asab-webui'
+    import { useForm } from 'react-hook-form'
```

Comment:
@petrKavulok I would add to the docu info about that the `react-hook-form` should be also installed and imported. Has it been tested on e.g. react-hook-form version 6 and version 7?